images/payload/hack/prepare.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -xe
+
+# Install the latest version of Docker
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
```

Comment:
Here you are assuming that the build will happen on linux, can you mention it somewhere?

Reply:
Where does `prepare.sh` referenced?